_data/navigation.js

```diff
@@ -140,6 +140,13 @@ module.exports = {
           },
         ],
       },
+      {
+        section: 'ADDITIONAL INFORMATION',
+        contents: [
+          { url: '/docs/node-operators-faq/', title: 'FAQ' },
+          { url: '/docs/glossary/', title: 'Glossary' },
```

Comment:
Let's call this page "Terminology" instead. Glossary is also accurate, but I think Terminology is more clear.

Reply:
We had a glossary before. It's very common in most blockchain documentation so people got used to looking for Glossary.